test/test_lagrangian_particle_tracking.jl

```diff
@@ -105,45 +106,45 @@ function run_simple_particle_tracking_tests(arch, timestepper; vertically_stretc
     vs = arch_array(arch, zeros(P))
     ws = arch_array(arch, zeros(P))
     ss = arch_array(arch, zeros(P))
+    ts = arch_array(arch, zeros(P))
 
     # Test custom constructor
-    particles = StructArray{TestParticle}((xs, ys, zs, us, vs, ws, ss))
+    particles = StructArray{TestParticle}((xs, ys, zs, us, vs, ws, ss, ts))
 
     velocities = VelocityFields(grid)
     u, v, w = velocities
     speed = Field(√(u*u + v*v + w*w))
-    tracked_fields = merge(velocities, (; s=speed))
+    tracked_fields = merge(velocities, (; s=speed, t=nothing))
 
     # Test second constructor
     lagrangian_particles = LagrangianParticles(particles; tracked_fields)
     @test lagrangian_particles isa LagrangianParticles
 
     model = NonhydrostaticModel(grid=grid, timestepper=timestepper,
-                                velocities=velocities, particles=lagrangian_particles)
+                                velocities=velocities, particles=lagrangian_particles, tracers=(:t, ))
```

Comment:
This is valid and preferred:

```suggestion
                                velocities=velocities, particles=lagrangian_particles, tracers=:t)
```